src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1782,6 +1782,24 @@ can be expressed as:
 
```

Comment:
Globally, I would rather separate the description from the instructions. For instance, export/import section may have a global description and then separate subsections.